encfs/FileUtils.cpp

```diff
@@ -1690,10 +1707,10 @@ bool unmountFS(EncFS_Context *ctx) {
     return false;
   }
 // Time to unmount!
-#if FUSE_USE_VERSION < 30
   fuse_unmount(ctx->opts->mountPoint.c_str(), nullptr);
-#else
-  fuse_unmount(fuse_get_context()->fuse);
+#ifdef __APPLE__
+  // fuse_unmount does not work on Mac OS, see #428
+  unmount(ctx->opts->mountPoint.c_str(), MNT_FORCE);
```

Comment:
Why MNT_FORCE?

Reply:
Same reason as above 😉

Reply:
And same confirmation as above. MacOS requires it.